### PR TITLE
Extend docs on RIS reader and writer (i.e. partly labeled RIS)

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR adds more information about using partly labeled data. 

To do:

- [x] add description on how to prepare a partly labeled dataset in RIS file format (@ottomattas )
- [x] add a screenshot of how the RIS file should look like (@ottomattas )

### CODE REVIEW

**Comment by ottomattas** [comment]
Hey @Rensvandeschoot , I see you have made a commit to docs/source/features/pre_screening.rst and docs/source/intro/datasets.rst
Are these the places you want this topic described further?

**Comment by Rensvandeschoot** [comment]
@ottomattas yes :-)

**Comment by ottomattas** [comment]
Instead of showing a screenshot, I linked a data set which shows partly labeled dataset in RIS format. (We reuse it for automatic testing so it is expected to stay up to date.)

**Comment by ottomattas** [comment]
This PR can now be merged.

File: docs/source/features/pre_screening.rst [hunk 1]
```diff
@@ -74,20 +74,35 @@ Select one of the :ref:`benchmark datasets <benchmark-datasets>`.
...
-and is empty for the records that you still need to screen using
-ASReview.
+A partly labeled dataset contains labeling decisions for part of the records,
+and part of the data does not have labels. The goal is to use the labels as
+prior knowledge to screen the unseen records. The labeled records will be
+recognized by ASReview LAB and are used to train the first iteration of the
+active learning model. This might be helpful if you switch from screening in
+another tool to screening with ASReview, or when updating an existing
+systematic review with more recent publications. For more options, see the
+`discussion board <https://github.com/asreview/asreview/discussions/975>`_.
```
Comment (J535D165) [inline docs/source/features/pre_screening.rst:84]: Google style guide prefers page title of the link.

```suggestion
systematic review with more recent publications. For more options, see `How to collaborate with ASReview? <https://github.com/asreview/asreview/discussions/975>`_ on the ASReview discussion board. 
```

File: docs/source/features/pre_screening.rst [hunk 2]
```diff
@@ -74,20 +74,35 @@ Select one of the :ref:`benchmark datasets <benchmark-datasets>`.
...
+For tabular datasets (:ref:`e.g., CSV, XLSX <data-format>`), the dataset should
+contain a column, called :ref:`label_included <column-names>` which is
+filled with 1's or 0's for the records that are already screened
+and is empty for the records that you still need to screen using ASReview.
+
+For the RIS file format, the dataset is handled automatically. The label 
+(ASReview_relevant, ASReview_irrelevant, ASReview_not_seen) is stored under the
+N1 (Notes) tag. If the N1 tag is missing, it will be created for each record
+after importing the dataset. You can find a representative demo dataset in our
+`GitHub repository <https://github.com/asreview/asreview/blob/master/tests/demo_data/baseline_tag-notes_labels.ris>`_
```
Comment (J535D165) [inline docs/source/features/pre_screening.rst:101]:
```suggestion
after importing the dataset. An example of a RIS file with N1 tag in the `ASReview
GitHub repository <https://github.com/asreview/asreview/blob/master/tests/demo_data/baseline_tag-notes_labels.ris>`_
```

File: docs/source/features/pre_screening.rst [hunk 3]
```diff
@@ -74,20 +74,35 @@ Select one of the :ref:`benchmark datasets <benchmark-datasets>`.
...
+  example in reference manager software.
+
+
+For tabular datasets (:ref:`e.g., CSV, XLSX <data-format>`), the dataset should
+contain a column, called :ref:`label_included <column-names>` which is
+filled with 1's or 0's for the records that are already screened
+and is empty for the records that you still need to screen using ASReview.
+
+For the RIS file format, the dataset is handled automatically. The label 
+(ASReview_relevant, ASReview_irrelevant, ASReview_not_seen) is stored under the
```
Comment (J535D165) [inline docs/source/features/pre_screening.rst:98]:
```suggestion
(`ASReview_relevant`, `ASReview_irrelevant`, `ASReview_not_seen`) is stored under the
```

**Comment by J535D165** [comment]
Thanks!

**Pull request merged**